nuxeo-platform-3d-core/src/main/java/org/nuxeo/ecm/platform/threed/importer/ThreeDImporter.java

```diff
@@ -78,12 +79,14 @@ protected String getModelFilename(final Blob zipContent) throws IOException {
         try {
             zipInputStream = new ZipInputStream(zipContent.getStream());
             while ((zipEntry = zipInputStream.getNextEntry()) != null) {
-                if (!zipEntry.isDirectory()) {
-                    if (SUPPORTED_EXTENSIONS.contains(FileUtils.getFileExtension(zipEntry.getName()))) {
-                        threeDFilename = FileManagerUtils.fetchTitle(zipEntry.getName());
-                        break;
-                    }
+                // skip if the entry is a directory, if it's not a supported extension or if it's hidden (by convention)
+                if (zipEntry.isDirectory() ||
+                    !SUPPORTED_EXTENSIONS.contains(FileUtils.getFileExtension(zipEntry.getName())) ||
+                    zipEntry.getName().startsWith(".")) {
```

Comment:
You might have some code style issues. static import should be before regular import and operand in an `if` should be on new line.